pathways/outputs.py

```diff
@@ -441,18 +442,24 @@ def print_totals_as_text(num_shipments, config, totals):
     print("Simulation results:")
     print("-----------------------")
     print("Avg. % shipments slipped: {0:.2f}%".format(totals.missing))
+    print("Total number of missed pests: {0:,.0f}".format(totals.total_missed_pests))
+    print(
+        "Total number of intercepted pests: {0:,.0f}".format(
+            totals.total_intercepted_pests
+        )
+    )
     print("Avg. infestation rate: {0:.3f}".format(totals.true_infestation_rate))
     if totals.max_missed_infestation_rate is not None:
         print(
-            "Max. infestation rate of slipped shipments: {0:.3f}\n"
-            "Avg. infestation rate of slipped shipments: {1:.3f}".format(
+            "Avg. infestation rate of slipped shipments: {0:.3f}\n"
+            "Max. infestation rate of slipped shipments: {1:.3f}".format(
```

Comment:
You can actually do:

```
"Max. infestation rate of slipped shipments: {totals.max_missed_infestation_rate:.3f}".format(**locals())
```

which would make these issues more obvious.

We could increase the minimal Python version, then this would be even simpler using an f-string:


```
f"Max. infestation rate of slipped shipments: {totals.max_missed_infestation_rate:.3f}"
```